README.md

```diff
@@ -1,64 +1,62 @@
-# climat_guardian
-Système de surveillance climatique 
+[Info]: <> (
+	All the information that are likely to need to be updated have a comment above them like this one
+)
+# Climat-Guardian
+This project was made for the non-profit organization memoires-informatiques\
+Memoires-informatiques has a lot of old computers and other electronic devices that need to be stored in a controlled environment\
+The goal of this project is to monitor the temperature and humidity of the different rooms of the organization and to display the data on a web interface
 
-## Postgresql
-### Installation
-This project uses a postgresql database wich is run in a docker container
-Start by creating a database called `memoires-info`
-Then to create the tables, you can run the file `init.sql` in the backend folder
+## Installation
+First thing you need to do is to clone the repository from the github page and install the dependencies
 ```bash
-psql -h localhost -U postgres -d memoires-info -a -f backend/init.sql
+git clone https://github.com/museebolo/climat-guardian.git
+cd climat-guardian/login
+composer install
+cd ../Interface
+npm install
+cd ..
 ```
-### Launch
-Once the database is created, you can start the docker container in the backend folder
-This will run the database on port 5432
-This also run adminer on port 8080 wich is a web interface to manage the database
-This also run a postg rest server on port 3000
+
+---
+[Info]: <> (
+	This is the list of all the files that end with .example
+)
+Once the repository is cloned you can remove the `.example` at the end of the following files:
+- login/.env.example
+- Interface/src/contexts/SampleContext.example.tsx
+- esp32/config/secrets.example.yaml
+
+---
+Once done you want to get the ip address of the computer that will run the docker containers
```

Comment:
Is IP address mandatory or can we use a host name?

Reply:
I've changed it and it now use a host name